repository-data/development/src/main/resources/hcm-config/hst/hosts.yaml

```diff
@@ -19,6 +19,7 @@ definitions:
             hst:homepage: root
             hst:mountpoint: /hst:hst/hst:sites/common
             jcr:primaryType: hst:mount
+          hst:scheme: http
```

Comment:
Is it worth adding a comment: # No SSL in local dev environment for simplicity

Reply:
Can't comment in YAML, see above